fabric-levelgenerators-v1/src/main/java/net/fabricmc/fabric/mixin/levelgenerator/MixinLevelGeneratorType.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.levelgenerator;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+import net.minecraft.world.level.LevelGeneratorType;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Mixin(LevelGeneratorType.class)
+public class MixinLevelGeneratorType {
+	@Shadow
+	@Final
+	private String name;
+
+	/**
+	 * @author Dinolek
+	 * @reason Replace ":" with "." in identifier
+	 */
+	@Overwrite
```

Comment:
Overwrite?

Reply:
Overwrite is discouraged, and I’ve had success using lang jsons in the past

Reply:
Oh wait this is for identifier

Reply:
I’ll take a look at the code again for more context

Reply:
I see. You’re changinng the ":" to "." for translation keys. Seems ok.

Reply:
How can I do it better?
Injecting before return seems stupid here.

Reply:
Could this happen before it even hits the `LevelGeneratorType`? would remove the need for this mixin?